src/features/inlay_hints.zig

```diff
@@ -353,7 +401,46 @@ fn writeNodeInlayHint(
         .aligned_var_decl,
         => {
             if (!builder.config.inlay_hints_show_variable_declaration) return;
-            try writeVariableDeclHint(builder, node);
+            const var_decl = builder.handle.tree.fullVarDecl(node) orelse return;
```

Comment:
```suggestion
            const var_decl = builder.handle.tree.fullVarDecl(node).?;
```
The same goes for the cases below here.